README.md

```diff
@@ -38,7 +54,12 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```
+Global scope is the largest scope in JavaScript.  Variables declared here can be accessed and changed
+by anything regardless of their position.  When you declare a new function or object any variables
+defined there can only be accessed by that function or object.  If you were to declare another
+function within a function, variables declared are only available within that inner function, but it has access
+to the previous functions variables as well as global variables.
 ```
```

Comment:
functions are unique in that they can create their own local scope.

How to declare a global variable vs local variable? i.e.:

`var x = 0 // local variable`
 `y = 1 //in the global scope`